public/app/plugins/panel/gettingstarted/GettingStarted.tsx

```diff
@@ -137,34 +79,114 @@ export class GettingStarted extends PureComponent<PanelProps, State> {
   };
 
   render() {
-    const { checksDone } = this.state;
+    const { checksDone, currentStep, steps } = this.state;
+
     if (!checksDone) {
       return <div>checking...</div>;
     }
 
+    const styles = getStyles();
+    const step = steps[currentStep];
+
     return (
-      <div className="progress-tracker-container">
-        <button className="progress-tracker-close-btn" onClick={this.dismiss}>
-          <Icon name="times" />
-        </button>
-        <div className="progress-tracker">
-          {this.steps.map((step, index) => {
-            return (
-              <div key={index} className={step.done ? 'progress-step completed' : 'progress-step active'}>
-                <a className="progress-link" href={step.href} target={step.target} title={step.note}>
-                  <span className="progress-marker">
-                    <Icon name={step.icon} size="xxl" />
-                  </span>
-                  <span className="progress-text">{step.title}</span>
-                </a>
-                <a className="btn-small progress-step-cta" href={step.href} target={step.target}>
-                  {step.cta}
-                </a>
-              </div>
-            );
-          })}
+      <div className={styles.container}>
+        <div className={styles.dismiss}>
+          <div onClick={this.dismiss}>Remove this panel</div>
         </div>
+        {currentStep === steps.length - 1 && (
+          <div className={cx(styles.backForwardButtons, styles.previous)} onClick={this.onPreviousClick}>
+            <Button icon="angle-left" variant="secondary" />
+          </div>
+        )}
+        <div className={styles.content}>
+          <Step step={step} />
+        </div>
+        {currentStep < steps.length - 1 && (
+          <div className={cx(styles.backForwardButtons, styles.forward)} onClick={this.onForwardClick}>
+            <Button icon="angle-right" variant="secondary" />
+          </div>
+        )}
       </div>
     );
   }
 }
+
+const getStyles = stylesFactory(() => {
+  const { theme } = config;
+  const backgroundImage = theme.isDark
+    ? 'public/img/Onboarding_Panel_dark.svg'
+    : 'public/img/Onboarding_Panel_light.svg';
```

Comment:
You don't have to put the images in the public dir anymore. You can put them next to the component and then:

```
import imageLight from './imageLight.svg';
import imageDark from './imageDark.svg';
// ...
// styles
background-image: url(${theme.isLight ? imageLight : imageDark});
```

Reply:
Nice!